src/main/kotlin/com/chattriggers/ctjs/minecraft/libs/ChatLib.kt

```diff
@@ -326,6 +322,17 @@ object ChatLib {
         return hist
     }
 
+    /**
+     * Adds a message to the player's chat history. This allows the message to
+     * show up for the player when pressing the up/down keys while in the chat gui
+     *
+     * @param message the message to add to chat history
+     */
+    @JvmStatic
+    fun addToChatHistory(message: String) {
+        Client.getChatGUI()!!.addToSentMessages(message)
+    }
```

Comment:
How about we add an index parameter for a bit more flexibility
```suggestion
    @JvmStatic
    @JvmOverloads
    fun addToChatHistory(message: String, index: Int? = null) {
        val sentMessages = Client.getChatGUI()!!.getSentMessages()
        sentMessages.add(index ?: sentMessages.size, message)
    }
```
Also, how about `addToSentMessageHistory`, that seem a bit less ambiguous.